go2_robot_sdk/go2_robot_sdk/infrastructure/ros2/ros2_publisher.py

```
@@ -256,19 +262,90 @@ def _lidar_worker(self) -> None:
+                                        ),
+                                    )
+                                    if (
+                                        points is not None
+                                        and not self._lidar_python_decode_fallback_logged
+                                    ):
+                                        self.node.get_logger().info(
+                                            "LiDAR decode fallback ACTIVE: using Python WASM decoder after C++ failure"
+                                        )
+                                        self._lidar_python_decode_fallback_logged = True
+                                except Exception:
+                                    pass
```

Ruff (0.14.8)

`try`-`except`-`pass` detected, consider logging the exception

(S110)